### PR TITLE
DF-921: Integrated logstash config and business logger from shared-library

### PR DESCRIPTION
**Pull request by leanoo**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -19,6 +19,9 @@ phonenumber_version=+
 oracle_version=12.1.0.1
 springfox_swagger2_version=2.9.2
 
+alv_shared_library_version=0.0.1-build-74
```
Comment (birom) [inline gradle.properties]: Please upgradle to the latest version and remove the redundant
```
alv:
    logstash:
        enabled: false
```
sections.

Comment (leanoo) [reply gradle.properties]: done

File: gradle.properties [hunk 2]
```diff
@@ -19,6 +19,9 @@ phonenumber_version=+
 oracle_version=12.1.0.1
 springfox_swagger2_version=2.9.2
 
+alv_shared_library_version=0.0.1-build-77
```
Comment (birom) [inline gradle.properties]: Please update to version 0.0.1-build-99. thx

Comment (leanoo) [reply gradle.properties]: done

**Pull request merged**